### PR TITLE
fix error from npcCreate()

### PR DESCRIPTION
**Pull request by StrawWagen**
added 2 lines, tested, and it works.

### CODE REVIEW

File: lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua [hunk 1]
```diff
@@ -110,13 +110,15 @@ e2function void entity:npcSetGlobalSquad( string squadIn )
...
 
-e2function entity npcCreate( string npcClass, vector position )
+e2function entity npcCreate( string npcClass, vector positionIn )
     if not hasAccess( self.player, 1 ) then return NULL end
     
     local npc = ents.Create( npcClass )
     
     if not isValidNpc( npc ) then return NULL end
     
+    position = Vector( positionIn[1], positionIn[2], positionIn[3] )
```
Comment (brandonsturgeon) [inline lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua:120]:
```suggestion
e2function entity npcCreate( string npcClass, vector position )
    if not hasAccess( self.player, 1 ) then return NULL end
    position = Vector( position[1], position[2], position[3] )

    local npc = ents.Create( npcClass )
    if not isValidNpc( npc ) then return NULL end
```

File: lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua [hunk 2]
```diff
@@ -141,6 +143,8 @@ e2function entity npcCreate( string npcClass, vector position, string modelIn )
     
     npc:SetKeyValue( "model", modelIn )
     
+    position = Vector( positionIn[1], positionIn[2], positionIn[3] )
```
Comment (brandonsturgeon) [inline lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua:146]: Same comment as the last one

**Review by brandonsturgeon — CHANGES_REQUESTED** [review]
Just a quick fixup and it's good to go 👍

**Comment by StrawWagen** [comment]
latest version has been tested, works without errors

**Comment by brandonsturgeon** [comment]
FYI you were previously defining `position` as a global variable, but the refactor solved that indirectly

File: lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua [hunk 3]
```diff
@@ -110,11 +110,11 @@ e2function void entity:npcSetGlobalSquad( string squadIn )
     this:SetKeyValue( "squadname", squadIn )
 end
 
-e2function entity npcCreate( string npcClass, vector position )
+e2function entity npcCreate( string npcClass, vector positionIn )
     if not hasAccess( self.player, 1 ) then return NULL end
+    position = Vector( positionIn[1], positionIn[2], positionIn[3] )
```
Comment (brandonsturgeon) [inline lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua:115]: If you really prefer `positionIn` over `position`, then this needs to be a local variable

```suggestion
    local position = Vector( positionIn[1], positionIn[2], positionIn[3] )
```

File: lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua [hunk 4]
```diff
@@ -132,15 +132,15 @@ e2function entity npcCreate( string npcClass, vector position )
...
     
     local npc = ents.Create( npcClass )
-    
     if not isValidNpc( npc ) then return NULL end
     
     npc:SetKeyValue( "model", modelIn )
-    
+
```
Comment (brandonsturgeon) [inline lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua:143]:
```suggestion
```

File: lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua [hunk 5]
```diff
@@ -132,15 +132,15 @@ e2function entity npcCreate( string npcClass, vector position )
     return npc
 end
 
-e2function entity npcCreate( string npcClass, vector position, string modelIn )
+e2function entity npcCreate( string npcClass, vector positionIn, string modelIn )
     if not hasAccess( self.player, 1 ) then return NULL end
+    position = Vector( positionIn[1], positionIn[2], positionIn[3] )
```
Comment (brandonsturgeon) [inline lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua:137]:
```suggestion
    local position = Vector( positionIn[1], positionIn[2], positionIn[3] )
```

**Review by brandonsturgeon — CHANGES_REQUESTED** [review]

**Review by brandonsturgeon — APPROVED** [review]

**Pull request merged**